### ISSUE TITLE
Problem with gulp process

### ISSUE THREAD

**Issue by mspanish** [open-start/opentheme#1]
Hello this looks brilliant. I've installed all npm and ruby dependencies, adjusted the config.yml, etc - and am still getting this error:

$ gulp
[gulp] Using gulpfile c:\Users\Stacey\Documents\GitHub\101spanishverbs\gulpfile.
js
[gulp] Starting 'sass'...
[gulp] Starting 'jekyll-build'...
[gulp] Starting 'watch'...
[gulp] Finished 'watch' after 592 ms

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:980:11)
    at Process.ChildProcess._handle.onexit (child_process.js:771:34)

**Comment by mspanish** [comment]
My /101spanishverbs folder also contains everything from your repo, so  there should not be missing files or folders.

**Comment by manavsehgal** [comment]
Thanks for reporting @mspanish - I will look into this. In the meantime you can use the shell scripts in the /scripts folder for doing your builds. You will need to adjust the scripts for your environment.

**Comment by mspanish** [comment]
Thanks I will try that. I did trying installing the NPM modules 1 by 1, as when I ran the npm install again, I saw some errors - but even after reinstalling each 1, I still got the ENOENT error. Your theme is so beautiful and will work so perfectly for the educational site I'm building that I'm quite determined to make this work! Thanks again for the amazing stuff you've put together, must have taken quite a bit of time. Hope to see you build it up and monetize it as well, us programmers are mostly working from a labor of love :)

**Comment by mspanish** [comment]
I tried the bash script but the problem is that I need the Jekyll site compiled first, not just pushed to the repo. The gulp process that I can't get working is what compiles all the .md changes using Jekyll - and so w/out that working I'm not sure how to do this. Can I use the standard Jekyll syntax for that, bypassing gulp? I will look on the Jekyll site. Sorry but I'm a Jekyll noob.

**Comment by mspanish** [comment]
I now have your site compiled locally looking super, on localhost:4000!

I used the compile from Github pages documentation, bundle exec jekyll serve

I am on Windows 8 - and I had to do a ton of Ruby and gems troubleshooting to get Jekyll working on my OS - reinstalling Ruby 2.0, updating all gems, etc. I've not tried your gulpfile yet again but the errors from before may have been due to Windows, and nothing to do with your file.

**Comment by mspanish** [comment]
Hello the site looks great online as well, I had to alter the main.min.css path to remove the slash - my header.html file has this instead for the css link:

  <link rel="stylesheet" type="text/css" href="css/main.min.css">

The site is just beginning and I didn't alter your theme much yet (I'm working on a new header color, yours is lovely but I'd like to explore variations :)

http://mspanish.github.io/101spanishverbs/

**Comment by manavsehgal** [comment]
Many thanks for your persistence to make things work! Truly the gift a creative looks for, someone to benefit from the results and share some feedback... which you did (smiles).

Sorry to see you struggle with Jekyll. Windows setup for anything-ruby can be painful. Glad to see you got your site up. If you continue to struggle with Windows you may want to try Cloud9 (https://c9.io). I use Cloud9 for hobby and work. They take away the setup pain and you only deal with a browser. Bash scripts will work on unix (Cloud9 setup offers unix-ubuntu + ruby + bundler all pre-installed).

"todo lo mejor" with 101SpanishVerbs! Maybe I will learn some Spanish now (smiles).

**Comment by mspanish** [comment]
Ah gracias! Actually that is just 1 of the sites I've used your theme on. The main site where I'll be using a lot more aspect of your theme is here: http://learningdecks.com

I've made some adjustments to the blog layout - and sort of created a showcase grid page based on your embed.html, I create my own data file and changed up some of the fields. Very nifty way to get a vcard type layout for a category, with very little work :)

I'm sure I"ll change more things as I keep going - I was looking at adding some different header colors, but got a bit lazy as I have tons of old flash content I'm trying to get online (while I still can, seems to me Flash is already nearly buried everywhere except e-learning!).

Thanks again for the large amount of work you've put into your theme. I'd be happy to offer any of the source files for the customizations I do.

Check out my Spanish page - there are lot of old but cool things there!
